routes/chat.js

```diff
@@ -32,6 +32,12 @@ router.get(
 router.get('/channels/:channelId', auth.isAuthAnonim, chatController.getChannel);
 router.post('/init-chat', auth.isAuth, chatController.initChat);
 router.post('/init-chat-anonymous', auth.isAuthAnonim, chatController.initChatAnonymous);
+router.post(
+  '/move-to-anon',
+  auth.isAuthAnonim,
+  validate(ChatValidation.moveToAnon),
+  chatController.initChatSourceAnonymous
+);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new route `/move-to-anon` has been added, which requires authentication as an anonymous user and validates the request payload using the `moveToAnon` schema from `ChatValidation`. The `initChatSourceAnonymous` controller function is invoked for this route. This looks good as long as the `auth.isAuthAnonim`, `validate(ChatValidation.moveToAnon)`, and `chatController.initChatSourceAnonymous` functions are correctly implemented.
```diff
+ router.post(
+   '/move-to-anon',
+   auth.isAuthAnonim,
+   validate(ChatValidation.moveToAnon),
+   chatController.initChatSourceAnonymous
+ );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->